tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -375,6 +374,9 @@ $(JitDisasmBatchScript)
 
 $(IlasmRoundTripBatchScript)
 
+REM Allow precommands to override the ExePath
```

Comment:
@kouvel does this change fix an issue or is it just a cleanup?

Reply:
The new test r2r-crossgens the test assembly and wants to change the ExePath in the script such that the r2r assembly is run. I didn't see a way to do that. ExePath is defined after the precommands, this change moves it to before the precommands so that the precommands can override the default ExePath.